services/example-website/server/templates/components/content-body.marko

```diff
@@ -6,7 +6,12 @@ import renderSiteNewsletterMenu from "@parameter1/base-cms-marko-web-theme-monor
 import renderSiteIdxNewsletterMenu from "@parameter1/base-cms-marko-web-theme-monorail/utils/render-site-idx-newsletter-menu";
 
 $ const { content, blockName } = input;
-$ const { contentGatingHandler, contentMeterState, req } = out.global;
+$ const { contentGatingHandler, contentMeterState, contentIdxFormState = {}, req } = out.global;
+
+<!-- update contentIdxFormStateDisplayBase on the req cookies being present -->
+<!-- @todo figure out how to do this ahead of time within the content route if possible -->
+$ contentIdxFormState.displayForm = !Boolean(get(req, `cookies.${get(contentIdxFormState, 'cookie.name')}`));
+$ const { formId: idxFormId, displayForm: displayIdxForm, cookie: idxSuveyCookie } = contentIdxFormState;
```

Comment:
Cleanup -- this logic should not live in the content body for implementing sites. Likely we need to add a new middleware to support generating the access state, and that would need to be added to the monorail theme.

What conditions cause the content to be gated in this manner? How does this interact with IdentityX access level content gating (`canAccess`)?